src/main/java/org/broadinstitute/hellbender/utils/read/ReadUtils.java

```diff
@@ -323,6 +323,14 @@ public static boolean readHasMappedMate( final GATKRead read ) {
         return read.isPaired() && ! read.mateIsUnmapped();
     }
 
+    /**
+     * @param read read to check
+     * @return true if the read is paired and has a mapped mate, otherwise false
+     */
+    public static boolean readHasMappedMate( final SAMRecord read ) {
```

Comment:
thats a good question, just looking in this file alone it seems to be wholly inconsistent one way or another. I'm not going to bother changing it because it would imply that there was some principled reason for me to unify ALL the formatting in this file at least which I don't necessarily want to do in this PR.